.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,97 @@
+name: 'Bug Report（bug 提交）'
+about: 'Report a software bug on this repo.（提交本仓库的 Bug）'
+title: ''
+labels:
+ - bug
+assignees: 
+ - TechQuery
+
+body:
+ - type: markdown
+   attributes:
+     value: |
+       **Describe the bug（bug 描述）**
+       A clear and concise description of what the bug is.（用简单的语言描述 bug。）
+
+ - type: textarea
+   id: bug-description
+   attributes:
+     label: 'Describe the bug'
+     placeholder: 'A clear and concise description of what the bug is.'
+     description: '用简单的语言描述 bug。'
+
+ - type: markdown
+   attributes:
+     value: |
+       **To Reproduce（复现方式）**
+       Steps to reproduce the behavior（复现 bug 的步骤）:
+
+ - type: textarea
+   id: reproduction-steps
+   attributes:
+     label: 'Steps to Reproduce'
+     placeholder: |
+       Steps to reproduce the behavior:
+       1. Go to '...'
+       2. Click on '....'
+       3. See bug
+       4. ...
+     description: '复现 bug 的步骤。'
+
+ - type: markdown
+   attributes:
+     value: |
+       **Expected behavior（预期现象）**
+       A clear and concise description of what you expected to happen.（用简单的语言描述期待看到的现象。）
+
+ - type: textarea
+   id: expected-behavior
+   attributes:
+     label: 'Expected behavior'
+     placeholder: 'A clear and concise description of what you expected to happen.'
+     description: '用简单的语言描述期待看到的现象。'
+
+ - type: markdown
+   attributes:
+     value: |
+       **Screenshots（截图）**
+       If applicable, add screenshots to help explain your problem.（可以截图辅助描述。）
+
+ - type: input
+   id: has-screenshots
```

```suggestion
 - type: textarea
   id: screenshots
```
GitHub issue 只有富文本编辑框可以上传图片。